orttraining/orttraining/python/ort_trainer.py

```diff
@@ -16,6 +16,14 @@
 from .checkpointing_utils import list_checkpoint_files, get_checkpoint_name, CombineZeroCheckpoint
 import onnxruntime.capi.pt_patch
 
+import sys
+sys.path.append('/bert_ort/liqun/onnxruntime/onnxruntime/core/providers/nuphar/scripts')
```

Comment:
>liqun [](start = 27, length = 5)

same comment

Reply:
need to package the symbolic_shape_infer with training.